packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.styles.ts

```diff
@@ -353,9 +349,28 @@ export const getStyles: IStyleFunction<IDropdownStyleProps, IDropdownStyles> = p
     ],
     dropdownItemsWrapper: { selectors: { '&:focus': { outline: 0 } } },
     dropdownItems: [globalClassnames.dropdownItems, { display: 'block' }],
-    dropdownItem: [...dropdownItemStyle, itemSelectors()],
+    dropdownItem: [
+      ...dropdownItemStyle,
+      itemSelectors(),
+      {
+        selectors: {
+          [HighContrastSelector]: {
+            border: 'none',
+          },
+        },
+      },
```

Comment:
sorry for not noticing this earlier, i think this change should go in `itemSelector`